gapir/cc/function_table.h

```diff
@@ -49,20 +49,20 @@ class FunctionTable {
   inline Function* lookup(Id id);
 
  private:
-  // Map of the supported function ids to the actual function implementations
-  std::unordered_map<Id, Function> mFunctions;
+  // Array of the actual function implementations by function ID
```

Comment:
Maybe add in comment *why* it's good to use an array here, rather than e.g. a map.

Reply:
Done